udfs/cloudbuild.yaml

```diff
@@ -14,19 +14,143 @@
 #
 
 steps:
-###########################################################
-# Step 1: Run UDF Unit Tests
-###########################################################
-- name: python:3.8.0-slim
-  entrypoint: 'bash'
-  args: ['udfs/tests/run.sh', '--pip_install_before_run']
-  env: ['SHORT_SHA=$SHORT_SHA']
-###########################################################
-# Step 2: Deploy if all tests pass
-###########################################################
+  ###########################################################
+  # Step 0: Install npm packages
+  ###########################################################
+- name: 'node'
+  id: 'step_0_install_npm_packages'
```

Comment:
nit: remove `step_${num}_` prefix
reasoning:
1. cloud build already prefixes all logs w. `Step # ${num} -` and this will create stutter in the logs.
2. If a contributor add steps they  have to go in and update all these id numbers.

Reply:
Good point. Removed